tests/YahooWeatherClientTests.php

```diff
@@ -10,11 +10,314 @@
 
 namespace Th3Mouk\YahooWeatherAPI\Tests;
 
+require 'src/Query/Query.php';
```

Comment:
This problem has been solved when I modify the correct phpunit.xml file finally.